### ISSUE TITLE
Network Protocol Error when mining a log in the park

### ISSUE THREAD

**Issue by IllagerCaptain** [MisterCheezeCake/SkyblockTweaks#4]
This only occurs when SkyblockTweaks is installed.
<details>
  <summary>disconnect-2024-07-22_08.59.44-client.txt</summary>
  
````---- Minecraft Network Protocol Error Report ----
// All lines are down!
// Server bug reporting link: https://hypixel.net/bug-reports/create

Time: 2024-07-22 08:59:44
Description: Packet handling error

java.lang.NumberFormatException: For input string: "300k"
	at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(Unknown Source)
	at java.base/jdk.internal.math.FloatingDecimal.parseFloat(Unknown Source)
	at java.base/java.lang.Float.parseFloat(Unknown Source)
	at wtf.cheeze.sbt.utils.actionbar.ActionBarTransformer.extractDataAndRunTransformation(ActionBarTransformer.java:135)
	at wtf.cheeze.sbt.SkyBlockTweaks.lambda$onInitialize$0(SkyBlockTweaks.java:92)
	at net.fabricmc.fabric.api.client.message.v1.ClientReceiveMessageEvents.lambda$static$4(ClientReceiveMessageEvents.java:92)
	at net.minecraft.class_7594.handler$zjg000$fabric-message-api-v1$fabric_allowGameMessage(class_7594.java:1073)
	at net.minecraft.class_7594.method_44736(class_7594.java)
	at net.minecraft.class_634.method_43596(class_634.java:904)
	at net.minecraft.class_7439.method_43631(class_7439.java:25)
	at net.minecraft.class_7439.method_11054(class_7439.java:11)
	at net.minecraft.class_2600.method_11072(class_2600.java:27)
	at net.minecraft.class_1255.method_18859(class_1255.java:162)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:136)
	at net.minecraft.class_1255.method_5383(class_1255.java:121)
	at net.minecraft.class_310.method_1523(class_310.java:1240)
	at net.minecraft.class_310.method_1514(class_310.java:882)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(Unknown Source)
	at java.base/jdk.internal.math.FloatingDecimal.parseFloat(Unknown Source)
	at java.base/java.lang.Float.parseFloat(Unknown Source)
	at wtf.cheeze.sbt.utils.actionbar.ActionBarTransformer.extractDataAndRunTransformation(ActionBarTransformer.java:135)
	at wtf.cheeze.sbt.SkyBlockTweaks.lambda$onInitialize$0(SkyBlockTweaks.java:92)
	at net.fabricmc.fabric.api.client.message.v1.ClientReceiveMessageEvents.lambda$static$4(ClientReceiveMessageEvents.java:92)
	at net.minecraft.class_7594.handler$zjg000$fabric-message-api-v1$fabric_allowGameMessage(class_7594.java:1073)
	at net.minecraft.class_7594.method_44736(class_7594.java)

-- Incoming Packet --
Details:
	Type: clientbound/minecraft:system_chat
	Is Terminal: false
	Is Skippable: true
Stacktrace:
	at net.minecraft.class_2600.method_59803(class_2600.java:56)
	at net.minecraft.class_8673.method_60882(class_8673.java:136)
	at net.minecraft.class_8673.method_59807(class_8673.java:113)
	at net.minecraft.class_2600.method_11072(class_2600.java:32)
	at net.minecraft.class_1255.method_18859(class_1255.java:162)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:136)
	at net.minecraft.class_1255.method_5383(class_1255.java:121)
	at net.minecraft.class_310.method_1523(class_310.java:1240)
	at net.minecraft.class_310.method_1514(class_310.java:882)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Connection --
Details:
	Protocol: play
	Flow: CLIENTBOUND
	Server type: OTHER
	Server brand: Hypixel BungeeCord (2024.7.15.1) <- vanilla

-- Custom Server Details --
Details:
	Proxy Instance: chi-hp-bungee103
	Proxy Version: 2024.7.15.1

-- System Details --
Details:
	Minecraft Version: 1.21
	Minecraft Version ID: 1.21
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 21.0.3, Azul Systems, Inc.
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Azul Systems, Inc.
	Memory: 1266535352 bytes (1207 MiB) / 2394947584 bytes (2284 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 4
	Processor Vendor: AuthenticAMD
	Processor Name: AMD A10-9620P RADEON R5, 10 COMPUTE CORES 4C+6G
	Identifier: AuthenticAMD Family 21 Model 101 Stepping 1
	Microarchitecture: unknown
	Frequency (GHz): 2.50
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: AMD Radeon R5 Graphics
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MiB): 512.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 23.20.808.1536
	Memory slot #0 capacity (MiB): 4096.00
	Memory slot #0 clockSpeed (GHz): 2.40
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 4096.00
	Memory slot #1 clockSpeed (GHz): 2.40
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 13250.10
	Virtual memory used (MiB): 8762.79
	Swap memory total (MiB): 5632.00
	Swap memory used (MiB): 447.86
	Space in storage for jna.tmpdir (MiB): available: 806028.81, total: 937793.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 806028.81, total: 937793.00
	Space in storage for io.netty.native.workdir (MiB): available: 806028.81, total: 937793.00
	Space in storage for java.io.tmpdir (MiB): available: 806028.81, total: 937793.00
	Space in storage for workdir (MiB): available: 806028.81, total: 937793.00
	JVM Flags: 2 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4096M
	Fabric Mods: 
		architectury: Architectury 13.0.6
		cloth-config: Cloth Config v15 15.0.128
			cloth-basic-math: cloth-basic-math 0.6.1
		entityculling: EntityCulling 1.6.6
		fabric-api: Fabric API 0.100.7+1.21
			fabric-api-base: Fabric API Base 0.4.42+6573ed8cd1
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.67+b5597344d1
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.29+5bd9f1bcd1
			fabric-block-api-v1: Fabric Block API (v1) 1.0.22+0af3f5a7d1
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.10+6573ed8cd1
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.52+0af3f5a7d1
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.15+6573ed8cd1
			fabric-command-api-v1: Fabric Command API (v1) 1.2.49+f71b366fd1
			fabric-command-api-v2: Fabric Command API (v2) 2.2.28+6ced4dd9d1
			fabric-commands-v0: Fabric Commands (v0) 0.2.66+df3654b3d1
			fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.15+b5597344d1
			fabric-convention-tags-v1: Fabric Convention Tags 2.0.18+7f945d5bd1
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.4.2+c111832ad1
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.29+0af3f5a7d1
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.25+6a6dfa19d1
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 20.2.12+16c4ae25d1
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.0+6fc22b99d1
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.12+6fc22b99d1
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.10+e633f883d1
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.53+6ced4dd9d1
			fabric-item-api-v1: Fabric Item API (v1) 11.0.0+afdfc921d1
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.3+78017270d1
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.47+0af3f5a7d1
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.45+df3654b3d1
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.11+8f3583aed1
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.13+3f89f5a5d1
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.1+3f89f5a5d1
			fabric-message-api-v1: Fabric Message API (v1) 6.0.13+6573ed8cd1
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 2.0.0+fe474d6bd1
			fabric-networking-api-v1: Fabric Networking API (v1) 4.2.0+ab7edbacd1
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.1.13+d1321076d1
			fabric-particles-v1: Fabric Particles (v1) 4.0.2+6573ed8cd1
			fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.10+65089712d1
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.0.25+5bd9f1bcd1
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.3.0+0ae0b97dd1
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.6.5+48fb1586d1
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.68+df3654b3d1
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.48+73761d2ed1
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.6+b5597344d1
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.71+df3654b3d1
			fabric-rendering-v1: Fabric Rendering (v1) 5.0.5+df16efd0d1
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.2.1+d153f344d1
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.2.0+7e687b32d1
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.24+b5597344d1
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.82+b5597344d1
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.23+6573ed8cd1
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.1.15+3dccd343d1
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.0.12+6573ed8cd1
		fabric-language-kotlin: Fabric Language Kotlin 1.11.0+kotlin.2.0.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 2.0.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.24.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.8.1
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.8.1
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.3
		fabricloader: Fabric Loader 0.15.11
			mixinextras: MixinExtras 0.3.5
		ferritecore: FerriteCore 7.0.0
		firmament: Firmament 1.8.0
			io_ktor_ktor-client-content-negotiation-jvm: ktor-client-content-negotiation-jvm 2.3.0
			io_ktor_ktor-client-core-jvm: ktor-client-core-jvm 2.3.0
			io_ktor_ktor-client-encoding-jvm: ktor-client-encoding-jvm 2.3.0
			io_ktor_ktor-client-java-jvm: ktor-client-java-jvm 2.3.0
			io_ktor_ktor-client-logging-jvm: ktor-client-logging-jvm 2.3.0
			io_ktor_ktor-events-jvm: ktor-events-jvm 2.3.0
			io_ktor_ktor-http-jvm: ktor-http-jvm 2.3.0
			io_ktor_ktor-io-jvm: ktor-io-jvm 2.3.0
			io_ktor_ktor-serialization-jvm: ktor-serialization-jvm 2.3.0
			io_ktor_ktor-serialization-kotlinx-json-jvm: ktor-serialization-kotlinx-json-jvm 2.3.0
			io_ktor_ktor-serialization-kotlinx-jvm: ktor-serialization-kotlinx-jvm 2.3.0
			io_ktor_ktor-utils-jvm: ktor-utils-jvm 2.3.0
			io_ktor_ktor-websocket-serialization-jvm: ktor-websocket-serialization-jvm 2.3.0
			io_ktor_ktor-websockets-jvm: ktor-websockets-jvm 2.3.0
			jarviscci: Jarvis Common Config Index 1.1.3
			mm_shedaniel: Manningham Mills (shedaniel's fork) 2.4.1
			moulconfig: Moulconfig 1.0.0
			org_slf4j_slf4j-api: slf4j-api 1.7.36
		java: OpenJDK 64-Bit Server VM 21
		lithium: Lithium 0.12.7
		memoryleakfix: Memory Leak Fix 1.1.5
		minecraft: Minecraft 1.21
		modmenu: Mod Menu 11.0.1
			placeholder-api: Placeholder API 2.4.0-pre.2+1.21
		paragraphs: Paragraphs 1.1.0
		presencefootsteps: Presence Footsteps 1.10.0+1.21
			kirin: Kirin UI 1.19.0+1.21
		roughlyenoughitems: Roughly Enough Items 16.0.729
			error_notifier: Error Notifier 1.0.9
		skyblocker: Skyblocker 1.21.1+1.21
			com_logisticscraft_occlusionculling: occlusionculling 0.0.8-SNAPSHOT
			meteordevelopment_discord-ipc: discord-ipc 1.1
			moe_nea_neurepoparser: neurepoparser 1.5.0
			org_apache_commons_commons-math3: commons-math3 3.6.1
			org_eclipse_jgit_org_eclipse_jgit: org.eclipse.jgit 6.9.0.202403050737-r
		skyblocktweaks-fabric: Skyblock Tweaks 0.1.0-Alpha.2
		sodium: Sodium 0.5.11+mc1.21
		sound_physics_remastered: Sound Physics Remastered 1.21-1.4.5
		tooltipscroll: Tooltip Scroll 1.3.0+1.21
		yet_another_config_lib_v3: YetAnotherConfigLib 3.5.0+1.21-fabric
			com_twelvemonkeys_common_common-image: common-image 3.10.0
			com_twelvemonkeys_common_common-io: common-io 3.10.0
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1````
</details>

**Comment by MisterCheezeCake** [comment]
Should be fixed in https://github.com/MisterCheezeCake/SkyblockTweaks/commit/59a623dfd1009a05a0436591cbf34396cf524ec6 and 0.1.0-Alpha.3

**Comment by IllagerCaptain** [comment]
> Should be fixed in [59a623d](https://github.com/MisterCheezeCake/SkyblockTweaks/commit/59a623dfd1009a05a0436591cbf34396cf524ec6) and 0.1.0-Alpha.3

It is, thanks.
